### DISCUSSION TITLE
Prioritize Specific Folders in Explorer Plugin

### DISCUSSION THREAD

**Discussion by sstream17** [Flow-Launcher/Flow.Launcher#2851] [Ideas]
I keep many files and shortcuts in a specific directory, C:/shortcuts.

Often, I'll start typing in Flow Launcher and the only two results are

1. A web search for the string.
2. Then my shortcut from my directory.

At this point, I can set my shortcut as topmost in this query. This works well when I type exactly the same query. But I often want the shortcut to have priority over the web search with more or fewer characters than the exact query.
I then have to set every single shortcut in my folder as topmost in their respective queries.

I would prefer if there was a way to set the full directory as higher priority.

I don't want to increase the priority of the full Explorer plugin, because then many more files which I want to access less frequently take up space over other more common actions.
I've also tried to set the "Quick Access Links" in the Explorer plugin settings, but this doesn't seem to do anything to help my situation.

I want to be able to prioritize all files from a specific directory without forcing all files to have priority and without needing to do the extra work of setting topmost on every file in the directory.

**Comment by jjw24** [comment]
Would bumping the Explorer plugin priority higher than Web Search plugin help in this scenario?

**Reply by sstream17** [reply]
No. Increasing the Explorer plugin priority will put too many files above the web search, making it harder to use. Even though that technically does prioritize the files I want, it prioritizes too many extra files.

**Reply by sstream17** [reply]
The other thing too, it doesn't prioritize the specific files above other ones within the Explorer plugin. So I would still have to set topmost on every file in my directory so it can appear above other files I don't want as prioritized.
